screens/SCWTest.tsx

```diff
@@ -0,0 +1,165 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { thirdwebClient, thirdwebWallets } from "@utils/thirdweb";
+import React, { useCallback } from "react";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import { prepareTransaction } from "thirdweb";
+import { ethers5Adapter } from "thirdweb/adapters/ethers5";
+import { base } from "thirdweb/chains";
+import {
+  useActiveAccount,
+  useActiveWallet,
+  useConnect,
+  useDisconnect,
+  useSendTransaction,
+  useSetActiveWallet,
+  useSwitchActiveWalletChain,
+} from "thirdweb/react";
+
+import config from "../config";
+
+const MainScreen: React.FC = () => {
+  const { connect: thirdwebConnect } = useConnect();
```

Comment:
I see that we're renaming and mentioning specifically thirdweb connect throughout this PR. What other methodologies for connect do we have?

Reply:
thirdweb is used for all external wallets connection
we also have
- ephemeral wallets
- direct private key connection
- connect via desktop wallet (currently disabled)